Gruntfile.js

```diff
@@ -252,32 +252,39 @@ module.exports = function(grunt) {
       configFiles: {
         files: [ 'Gruntfile.js', 'package.json' ],
         options: {
-          reload: true
+          reload: true,
+          interval: 1000
```

Comment:
Huh, I presumed that wouldn't work because the documentation doesn't imply it does, and it also means that if you name one of your sections "options" it will silently not work.

Anyway, it does, and I've done it, and yay!